.gitignore

```diff
@@ -5,6 +5,7 @@ shipwright-build-controller
 build/_output
 build/_test
 hub-linux-*
+/waiter
```

Comment:
What is getting ignored here ?

Reply:
That's the binary produced by `ko`, for the record.

Reply:
Sorry, that's not generated by `ko`. I'm going to revert this change.